### ISSUE TITLE
feature request: progress bar

### ISSUE THREAD

**Issue by bittlingmayer** [facebookresearch/ParlAI#104]
It would be really helpful if the % and so on were printed out to the console during training, similar to fasttext.

Estimated time to finish could be good too.

**Comment by alexholdenmiller** [comment]
Hi @bittlingmayer, for what specifically would you like the progress bar? Do you mean the % portion until a model has finished training? If that's what you mean, then this is model-dependent and is up to the model to provide--many models will not necessarily be able to produce this number.

**Comment by ajfisch** [comment]
Also, as it is now, ParlAI randomly samples from the training sets vs shuffle + iterate -- as FastText does. So there isn't really a concept of total number of epochs. E.g. the DrQA example trains until the validation accuracy hasn't increased in more than N consecutive evaluations.

**Comment by jaseweston** [comment]
We will be adding a max-train-time flag soon though, and will log that. But then the time to finish is kind of clear at that point..
